### PR TITLE
"Element with lang attribute has valid language tag" [de46e4]: Updated Failed Examples 4 and 5 to reflect Applicability

### PR DESCRIPTION
**Pull request by giacomo-petri**
I've updated:
- Failed Example 4 
- Failed Example 5

to reflect the updated Applicability, including a reference to the [text inheriting its programmatic language][] definition in each failure.

Closes issue(s): #2063

Need for Call for Review:
This will require a 1 week Call for Review 

---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/element-lang-valid-de46e4.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ This `article` element has a `lang` [attribute value][] which consists of only [
 
 #### Failed Example 4
 
-The `lang` [attribute value][] does not have a valid language tag. The `lang` attribute must be valid because the content is [visible][].
+The `lang` [attribute value][] does not have a valid language tag. Even though its descendant `p` element is not [included in the accessibility tree][] due to the presence of the `aria-hidden="true"` attribute, the content of the `p` element is still [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```
Comment (Jym77) [inline _rules/element-lang-valid-de46e4.md]:
```suggestion
The `lang` [attribute value][] does not have a valid language tag. Even though the `p` element is not [included in the accessibility tree][] due to the presence of the `aria-hidden="true"` attribute, it is [visible][]; therefore its content is [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```
(there is only one `p` element, so no need to disambiguate, plus "its (descendant)" refers here to the `lang` attribute (value) (subject of the previous sentence) making "its descendant" technically incorrect)

File: _rules/element-lang-valid-de46e4.md [hunk 2]
```diff
@@ -200,7 +200,7 @@ The `lang` [attribute value][] does not have a valid language tag. The `lang` at
 
 #### Failed Example 5
 
-The `lang` [attribute value][] does not have a valid language tag, and its [descendant][] is not [visible][] though it is still [included in the accessibility tree][].
+The `lang` [attribute value][] does not have a valid language tag. Even though its descendant `p` element is not [visible][] due to the CSS properties set, the content of the `p` element is still [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```
Comment (Jym77) [inline _rules/element-lang-valid-de46e4.md]:
```suggestion
The `lang` [attribute value][] does not have a valid language tag. Even though the `p` element is not [visible][] due to the being moved off-screen, it is [included in the accessibility tree][]; therefore its content is [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
Minor polishing.

**Review by Jym77 — DISMISSED** [review]

**Review by colabottles — APPROVED** [review]

File: _rules/element-lang-valid-de46e4.md [hunk 3]
```diff
@@ -200,7 +200,7 @@ The `lang` [attribute value][] does not have a valid language tag. The `lang` at
 
 #### Failed Example 5
 
-The `lang` [attribute value][] does not have a valid language tag, and its [descendant][] is not [visible][] though it is still [included in the accessibility tree][].
+The `lang` [attribute value][] does not have a valid language tag. Even though the `p` element is not [visible][] due to the being moved off-screen, it is [included in the accessibility tree][]; therefore its content is [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```
Comment (dan-tripp-siteimprove) [inline _rules/element-lang-valid-de46e4.md]:
```suggestion
The `lang` [attribute value][] does not have a valid language tag. Even though the `p` element is not [visible][] due to it being moved off-screen, it is [included in the accessibility tree][]; therefore its content is [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```

**Review by dan-tripp-siteimprove — CHANGES_REQUESTED** [review]

**Review by dan-tripp-siteimprove — APPROVED** [review]

**Review by colabottles — APPROVED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Comment by giacomo-petri** [comment]
This PR is in call for review for 1 week. Review ends Sept 22nd.

**Comment by carlosapaduarte** [comment]
Call for review ended: merging

**Pull request merged**